vxportia/router.py

```diff
@@ -0,0 +1,92 @@
+
+from vumi.errors import DispatcherError
+from vumi.dispatchers.base import BaseDispatchRouter
```

Comment:
All of the router implementations I'm finding build off of this `BaseDispatchRouter` class though? Are you saying I'm supposed to be subclassing the entirely different `vumi.dispatchers.endpoint_dispatchers.Dispatcher` base class and build everything from there onwards instead of doing a Router implementation?